### PR TITLE
Expose organization contract interact using mosaic.js

### LINKED ISSUE #130: Expose the Organization Contract Interact from Mosaic.js directly

### PR DESCRIPTION
**Pull request by abhayks1**
PR covers below:
- [x] Organization contract interact being exposed from mosaic.js directly
- [x] Readme update.
- [x] Test updates.

Fixes #130

### CODE REVIEW

File: test/integration/DelayedRecovery.js [hunk 1]
```diff
@@ -313,15 +311,15 @@ describe('Delayed Recovery', async () => {
     const mockToken = mockTokenDeployerInstance.addresses.MockToken;
 
     const organizationOwnerAddress = deployerAddress;
-    const { Organization } = Mosaic.ContractInteract;
+    const { Organization } = Package;
```
Comment (schemar) [inline test/integration/DelayedRecovery.js]: shouldn't you instead require the organization in the head? Same in other test files.

Comment (abhayks1) [reply test/integration/DelayedRecovery.js]: Done!

File: index.js [hunk 2]
```diff
@@ -11,11 +11,13 @@ const Rules = require('./lib/setup/Rules');
 const PricerRule = require('./lib/helper/rules/PricerRule');
 const GnosisSafe = require('./lib/helper/GnosisSafe');
 const Recovery = require('./lib/ContractInteract/Recovery');
+const Mosaic = require('@openstfoundation/mosaic.js');
 
 const SignEIP1077Extension = require('./utils/SignEIP1077Extension');
 new SignEIP1077Extension();
 
 module.exports = {
+  Organization: Mosaic.ContractInteract.Organization,
```
Comment (schemar) [inline index.js]: Shouldn't this be under `Contracts`? 🤔

Comment (abhayks1) [reply index.js]: Yes! Updating it to be under ContractInteract 👍 
We have ContractInteract changes in the other PR.

Comment (abhayks1) [reply index.js]: Travis is green 🚀

**Pull request merged**